src/en/charms.md

```diff
@@ -75,7 +75,7 @@ background.
 
 As we said at the beginning, we are going to scale up this application to cope 
 with a lot of traffic. To do that we will use a common TCP/HTTP load balancing 
-service, HAProxy, so we should also deploy a charm for that now:
```

Comment:
I believe this is one of the ones we left previously, since it is valid to call it a load-balancing service, but I guess we can also refer to it as an application